src/harness/testcases/testdata/pal_db_record_0.json

```diff
@@ -0,0 +1,31 @@
+   {  
```

Comment:
Is this actually used?

Reply:
this is an old file asked to be removed. We removed this file for now.